pJanks/jeopardy
# New instantiations Update DOM with categories

### What's this PR do?<br>
Updates DOM with random categories.
Instantiate new Round, Game, & Players upon clicking "Start Game" button.<br>
### Where should the reviewer start?<br>
Try entering player names and check categories<br>
### How should this be manually tested?<br>
Try refreshing the page multiple times to ensure categories are randomly generated.<br>
### Potential changes?<br>
Need to re-style category headings.
